pinaxcon/templates/static_pages/about/north_bay_python.html

```diff
@@ -9,29 +9,47 @@
+<p>North Bay Python is a single-track conference, with a carefully curated set of talks representing the truly diverse Python community and their different areas of interest.
+<p>
+
+<p>If a topic is less to your interest, or you've met some people you really want to sit down and chat with, we'll have plenty of areas away from the main theatre to catch up and chat as well.</p>
+
+<h2>Petaluma, California</h2>
+
+<p>North Bay Python's home is Petaluma, a delightfully quaint dairy town, nestled on a river at the southern edge of California's Wine Country. We've got beautiful scenery right on our doorstep, and we're less than an hour's drive from San Francisco over the Golden Gate Bridge.</p>
+
+<p>The Mystic is not the only local example of early 1900s architecture, either; downtown is full of historic buildings, and it's at the center of the maker movement and a thriving craft brewery scene.</p>
+
+<h3>Getting here</h3>
+
+<h4>By car</h4>
+
+<p>If you're driving up, Downtown Petaluma is at exit 472 on Highway 101, 35 miles north of the Golden Gate Bridge. All parking is free in Petaluma, including in the undercover garages at Keller St and Theatre Square, both are in short walking distance of the Mystic.</p>
+
+<h4>By bus</h4>
+
+<p>Public transit to Petaluma is not great. You can take the 101 bus operated by Golden Gate Transit from downtown San Francisco, or south from Santa Rosa. Depending on sponsorship, we hope to run a free shuttle with BART and Caltrain connections for people from further out of town.</p>
+
+<h4>By plane</h4>
 
-<h2>The Location</h2>
+<p>If you're coming from out of the area, you may want to consider Sonoma County Airport (STS) &ndash; it's 30 minutess out of Petaluma, and has nonstop flights to most major west coast cities. If you can't make it to STS, you can also try San Francisco (SFO) or Oakland (OAK) international airports.</p>
 
-<p>Petaluma is right in the middle of the beautiful scenery of the North Bay, and less than an hour's drive from San Francisco over the Golden Gate Bridge. The Mystic is not the only example of early 1900s architecture, either; downtown is full of historic buildings, as well as being at the center of thing things like the maker movement and a thriving craft brewery scene.</p>
+<h3>Staying here</h3>
 
-<p>If you want to stay overnight, we have deals with local hotels - the closest one is just one block away. If you're driving up, Petaluma has plenty of free parking, and if you don't feel like driving, we'll be putting on a shuttle from public transport stops as well.</p>
+<p>Petaluma, as well as being a place where people live, also has hotels! We're arranging deals with some of the best local hotels in the area &ndash; the closest is just one block away. We'll share details of our hotel deals with you when conference tickets go on sale.</p>
```

Comment:
drop the "as well" aside, it's nice but unwieldy. "share details of our hotel deals" -> "share details"